src/openrct2/actions/GameSetSpeedAction.h

```diff
@@ -0,0 +1,33 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2023 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "GameAction.h"
+
+class GameSetSpeedAction final : public GameActionBase<GameCommand::SetGameSpeed>
+{
+private:
+    int32_t _speed{ 1 };
+
+public:
+    GameSetSpeedAction() = default;
```

Comment:
Do we need this to compile somewhere? Otherwise I would remove it, as we never use the default ctor and would technically always want to use the action with a given speed.

Reply:
I am not a C++ expert be any means... I only made the .h file because the other game actions did. If it's not needed, then yeah let's remove it.